package.json

```diff
@@ -21,9 +24,9 @@
     "@types/react": "^15.0.0",
     "@types/react-dom": "^0.14.20",
     "@types/node": "^7.0.5",
-    "@types/whatwg-fetch": "0.0.33",
     "awesome-typescript-loader": "^3.0.0-beta.18",
     "source-map-loader": "^0.1.5",
-    "typescript": "^2.1.4"
+    "typescript": "^2.1.4",
+    "uglify-js": "github:mishoo/UglifyJS2#harmony"
```

Comment:
Can we not add a dev dependency through NPM by doing this?

`npm install uglify-js --save-dev`